### PR TITLE
Fix various token_send QTreeWidget layout issues

### PR DESCRIPTION
**Pull request by Andrew-128**
- Standardized the heights and surrounding margins of the various in-table widgets
- Fixed the hover-highlight effect on table rows (visible for me only on the Dark theme)
- Simplified some conditional statements
- Make the Clear button on the minting form work

### CODE REVIEW

File: electroncash_gui/qt/token_send.py [hunk 1]
```diff
@@ -842,7 +860,9 @@ def clear_form(self):
         self.amount_e.clear()
         if self.form_mode == self.FormMode.send:
             self.te_payto.clear()
-        self.te_desc.clear()
+        elif self.form_mode == self.FormMode.mint:
+            self.te_desc.clear()
+        self.nfts_to_mint.clear()
```
Comment (cculianu) [inline electroncash_gui/qt/token_send.py:865]: I think you reversed the 2 lines here....?

File: electroncash_gui/qt/token_send.py [hunk 2]
```diff
@@ -242,7 +243,7 @@ def sort_func(u):
         hbox = QtWidgets.QHBoxLayout()
         vbox_bottom.setStretch(vbox.count()-1, 100)
         hbox.setContentsMargins(0, 0, 0, 0)  # No inset
-        # Additional BCH to Send
+        # Additional BCH to SendF
```
Comment (cculianu) [inline electroncash_gui/qt/token_send.py:246]: typo

File: electroncash_gui/qt/token_send.py (hunk 1 above)
Comment (Andrew-128) [reply electroncash_gui/qt/token_send.py]: Gah, yes.  Sorted.

File: electroncash_gui/qt/token_send.py (hunk 2 above)
Comment (Andrew-128) [reply electroncash_gui/qt/token_send.py]: Thx, fixed.

**Comment by cculianu** [comment]
Looks better on macOS too. Much improved.

**Pull request merged**